packages/@uppy/xhr-upload/src/index.js

```diff
@@ -44,7 +46,7 @@ function setTypeInBlob (file) {
 }
 
 module.exports = class XHRUpload extends Plugin {
-  static VERSION = require('../package.json').version
+  static VERSION = version
```

Comment:
I don't think that's correct, I'm afraid it would break https://github.com/andrewimm/babel-plugin-inline-package-json.